eqllib/analytics/execution/T1170-koadic-grouped-behavior-mshta.toml

```diff
@@ -0,0 +1,24 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies an unordered combination of events tied to Koadic's Mshta stager."
+id = "8c238e5c-7f40-4958-b046-1e048709c9b8"
+name = "Koadic Grouped Behavior (Mshta Stager)"
+os = ["windows"]
+tactics = ["Execution", "Defense Evasion", "Command and Control"]
+tags = ["koadic"]
+techniques = ["T1170","T1085"]
+updated_date = "12/04/2019"
+references = []
+
+[analytic]
+query = '''
+join
```

Comment:
are these ordered chronologically at all? it looks a little bit like that, so I'm wondering if a sequence makes more sense.
also is there any thing we can join _by_? user_name? user_sid?